src/lib/logMsg/logMsg.h

```diff
@@ -1793,13 +1802,77 @@ inline void LM_TRANSACTION_START(const char* keyword, const char* ip, int port,
 
 /* ****************************************************************************
 *
+* LM_TRANSACTION_SET_SRV -
+*/
+inline void LM_TRANSACTION_SET_SRV(const std::string& _srv)
+{
+  if (_srv != "")
+  {
+    snprintf(srv, sizeof(srv), _srv.c_str());
+  }
+  else
+  {
+    snprintf(srv, sizeof(srv), "<default>");
```

Comment:
"N/A" is for places in which srv is not applicable (e.g. traces during Orion startup). "<default>" is for places where a srv makes sense but the client doesn't provide it (e.g. he/she doesn't sent the Fiware-Service header).